src/modules/property-detail/routes/single-property.route.tsx

```diff
@@ -25,7 +27,7 @@ export function SinglePropertyRoute() {
   const { data: bookings, isLoading: bookingLoading } = useGetBookings();
   const [checkInDate, setCheckInDate] = useState<Dayjs | null>(null);
   const [checkOutDate, setCheckOutDate] = useState<Dayjs | null>(null);
-
+  const { data: user } = useAuth();
```

Comment:
You could also use `const user = useAppSelector((state) => state.auth.user);` as this component is already wrapped by AuthLayout component which calls this hook